record/menubar/FlashDateSeparators/true.js

```diff
@@ -0,0 +1,55 @@
+const aperture = require('aperture')()
+const delay = require('delay')
+const robot = require('robotjs')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+const { makeAppActive, moveAndResizeApp, compressVideo } = require('../../utils')
+
+module.exports = {
+  run: async (outputPath) => {
+    console.log('> Recording menu bar clock FlashDateSeparators with param set to true')
+    // Set the menu bar menuExtras to only show the clock, it will be on the left of notification center, siri, and spotlight search.
+    const { stderr: setEnvError } = await exec(`defaults write com.apple.systemuiserver menuExtras "('/System/Library/CoreServices/Menu Extras/Clock.menu')" && defaults write com.apple.menuextra.clock FlashDateSeparators -bool true && killall SystemUIServer && sleep 10`)
```

Comment:
`menuExtras` does not seem to work on BigSur.

```suggestion
    const { stderr: setEnvError } = await exec(`defaults write com.apple.menuextra.clock FlashDateSeparators -bool true && killall SystemUIServer && sleep 10`)
```